applications/StructuralMechanicsApplication/tests/TimoshenkoBeams/2D3N_curved/timoshenko_curved_beam_2d3N_test_parameters.json

```diff
@@ -0,0 +1,138 @@
+        "displacement_absolute_tolerance"      : 1e-9,
+        "residual_relative_tolerance"          : 1.0e-5,
+        "residual_absolute_tolerance"          : 1e-7,
+        "max_iteration"                        : 10,
+        "use_old_stiffness_in_first_iteration" : false,
+        "rotation_dofs"                        : true,
+        "volumetric_strain_dofs"               : false
+    },
+    "processes"        : {
+        "constraints_process_list" : [{
+            "python_module" : "assign_vector_variable_process",
+            "kratos_module" : "KratosMultiphysics",
+            "process_name"  : "AssignVectorVariableProcess",
+            "Parameters"    : {
+                "model_part_name" : "Structure.DISPLACEMENT_Displacement_Auto1",
+                "variable_name"   : "DISPLACEMENT",
+                "interval"        : [0.0,"End"],
+                "constrained"     : [true,true,true],
+                "value"           : [0.0,0.0,0.0]
+            }
+        },{
+            "python_module" : "assign_vector_variable_process",
+            "kratos_module" : "KratosMultiphysics",
+            "process_name"  : "AssignVectorVariableProcess",
+            "Parameters"    : {
+                "model_part_name" : "Structure.ROTATION_Rotation_Auto1",
+                "variable_name"   : "ROTATION",
+                "interval"        : [0.0,"End"],
+                "constrained"     : [true,true,true],
+                "value"           : [0.0,0.0,0.0]
+            }
+        }],
+        "loads_process_list"       : [{
+            "python_module" : "assign_vector_by_direction_to_condition_process",
+            "kratos_module" : "KratosMultiphysics",
+            "check"         : "DirectorVectorNonZero direction",
+            "process_name"  : "AssignVectorByDirectionToConditionProcess",
+            "Parameters"    : {
+                "model_part_name" : "Structure.PointLoad2D_Load_on_points_Auto1",
+                "variable_name"   : "POINT_LOAD",
+                "interval"        : [0.0,"End"],
+                "modulus"         : "1000*t",
+                "direction"       : [0.0,-1.0,0.0]
+            }
+        }],
+        "list_other_processes"     : [],
+        "json_check_process" : [
+            {
+                "python_module"   : "from_json_check_result_process",
+                "kratos_module" : "KratosMultiphysics",
+                "help"                  : "",
+                "process_name"          : "FromJsonCheckResultProcess",
+                "Parameters"            : {
+                    "check_variables"              : ["DISPLACEMENT", "ROTATION"],
+                    "gauss_points_check_variables" : ["BENDING_MOMENT","SHEAR_FORCE", "AXIAL_FORCE"],
+                    "input_file_name"              : "TimoshenkoBeams/2D3N_curved/timoshenko_curved_beam_2d3N_test_results.json",
+                    "model_part_name"              : "Structure.Parts_Beam_Beam_Auto1",
```

Comment:
same

Reply:
I only maintained all nodes in the curved element test since there are only a few already and the coverage of the test is more relevant